### PR TITLE
refactor(ni): remove usage of arrow from NI parser

### PR DESCRIPTION
**Pull request by amv213**
## Issue

https://github.com/electricitymaps/electricitymaps-contrib/issues/6135

## Description

This PR removes the dependency on `arrow` from the NI parser.

### Double check

- [x] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: parsers/NI.py [hunk 1]
```diff
@@ -92,14 +90,13 @@ def extract_text(full_text: str, start_text: str, end_text: str | None = None):
...
-    # date format is: "'Actualizado: 07/07/2017 01:00:50 PM'"
-
-    datetime_text = extract_text(text, "Actualizado: ", "'")
-    datetime_arrow = arrow.get(datetime_text, "DD/MM/YYYY hh:mm:ss A")
-    datetime_datetime = arrow.get(datetime_arrow.datetime, TIMEZONE).datetime
-
-    return datetime_datetime
+def get_time_from_system_map(text: str) -> datetime:
+    # date format is: "'InformaciÃ³n en Tiempo Real al 02/04/2024 05:57:40 AM'"
+    datetime_text = extract_text(text, "en Tiempo Real al ", "'")
```
Comment (VIKTORVAV99) [inline parsers/NI.py:95]: Why did this datetime_text change?

**Review by VIKTORVAV99 — COMMENTED** [review]
Just one question really, otherwise things look good.

File: parsers/NI.py (hunk 1 above)
Comment (amv213) [reply parsers/NI.py:95]: They seem to have changed the subtitle on their webpage sometime between  [2023/06/02](https://web.archive.org/web/20230206194327/https://www.cndc.org.ni/graficos/MapaSIN/index.php) and [2023/11/28](https://web.archive.org/web/20231128101700/https://www.cndc.org.ni/graficos/MapaSIN/index.php)  👀

Comment (VIKTORVAV99) [reply parsers/NI.py:95]: Man arrow is filled with crazy unexpected stuff!

I'm glad we are removing it!

**Review by VIKTORVAV99 — APPROVED** [review]

**Pull request merged**